<?xml version="1.0" ?>
<launch>
	<node name="test_lagsDS_node" pkg="lagsDS" type="test_lagsDS_node" output="screen">
		<param name="model_path"                  value="$(find lagsDS)/models/coManip-DS-0/"/>
		<rosparam command="load" file="$(find lagsDS)/models/coManip-DS-0.yml"/>
	</node>
</launch>